docs/antrea-network-policy.md

```diff
@@ -540,6 +588,98 @@ The [ovs-pipeline doc](design/ovs-pipeline.md) contains more information on how
 policy rules are realized by OpenFlow, and how the priority of flows reflects the
 order in which they are enforced.
 
+## ClusterGroup
+
+A ClusterGroup (CG) CRD is a specification of how workloads are grouped together.
+It allows admins to group Pods using traditional label selectors, which can then
+be referenced in ACNP in place of stand alone `podSelector` and/or `namespaceSelector`.
```

Comment:
you use "stand alone" here but "stand-alone" earlier. I think you should use "stand-alone" or "standalone" consistently.

Reply:
done